client/components/payment-method-logos/index.tsx

```diff
@@ -71,11 +71,27 @@ export const WooPaymentMethodsLogos: React.VFC< {
 	maxElements: number;
 } > = ( { maxElements = 10 } ) => {
 	const totalPaymentMethods = 21;
-	const maxElementsMiniView = 5;
+	const [ maxShownElements, setMaxShownElements ] = useState( maxElements );
+
+	useEffect( () => {
+		const getShowElements = () => {
+			if ( window.innerWidth <= 480 ) {
+				setMaxShownElements( 5 );
+			} else if ( window.innerWidth <= 768 ) {
+				setMaxShownElements( 7 );
+			} else {
+				setMaxShownElements( maxElements );
+			}
+		};
+
+		getShowElements();
+		window.addEventListener( 'resize', getShowElements );
+	}, [ maxElements ] );
+
 	return (
 		<>
 			<div className="woocommerce-payments-method-logos">
```

Comment:
```suggestion
			<div className="connect-account-page__payment-methods--logos">
```
Since the component will be used on the connect page, I believe we should inherit the classes.

Reply:
changed in [6564ac2](https://github.com/Automattic/woocommerce-payments/pull/8977/commits/6564ac2b72c939d990a7ad759fb88aa2d8cef9d0) . thx